chapters/scoping.tex

```diff
@@ -759,13 +747,13 @@ \subsection{Generation of the flat equation system}\label{generation-of-the-flat
   conditional declaration expressions evaluate to true or it is a
   constant in a package.
   \begin{nonnormative}
-  Conditional components can be used in connect-statements, and if the component is conditionally disabled the connect-statement is removed.
+  Conditional components can be used in connect-equations, and if the component is conditionally disabled the connect-statement is removed.
   \end{nonnormative}
 \end{itemize}
 
-This leads to a flattened equation system, except for connect and transition statements. These have to be transformed as described in
+This leads to a flattened equation system, except for connect and \lstinline!transition! equations. These have to be transformed as described in
```

Comment:
I think connect should be treated similarly as transition here.

Reply:
Separate PR?

Reply:
Filed #2713 to settle this.  Enough to resolve?